content/guides/models/tables/visualize-tables.md

```diff
@@ -71,6 +71,71 @@ To view the two tables side-by-side, change the first dropdown from "Merge Table
 * **explore two tables independently**: scroll through and focus on the side/rows of interest
 
 
+
+## Visualize how values change throughout your runs
+
+View how values you log to a table change throughout your runs with a step slider. Slide the step slider to view the values logged at different steps. For example, you can view how the loss, accuracy, or other metrics change after each run. 
+
+Each call to `wandb.log()` in your Python script creates a value known as a step (`_step`). By default, W&B uses `_step` values as the units, or key, of the slider. 
```

Comment:
This is a little confusing. I think maybe it would be helpful to explain how the slider works (given any key) separately from what key it uses by default. Actually using the slider is probably easier than explaining it, just like board game instructions.